balancer-labs/frontend-v2
# Tooltips UI

# Description

Currently it can be hard to distinguish between tooltips and the background content, since there is not much contrast behind the tooltip. This PR:
- Adds a deeper shadow to light mode
- Adds a colorful radial gradient to dark mode.
- Adds a bottom and top triangle towards the tooltip trigger
- Adds a small upwards / downwards animation

## Type of change

- [x] Other: UI improvement

## How should this be tested?

View as many tooltips as possible to ensure none of them have broken and that the padding looks good.

## Visual context

<img width="816" alt="Screen Shot 2022-04-20 at 4 17 52 PM" src="https://user-images.githubusercontent.com/1121139/164264955-3050bc02-36fb-47bc-bfec-17b79b120a51.png">

## Checklist:
